lib/fs/basicfs_windows.go

```diff
@@ -190,6 +190,21 @@ func (f *BasicFilesystem) Lchown(name, uid, gid string) error {
 	return windows.SetSecurityInfo(hdl, windows.SE_FILE_OBJECT, si, (*windows.SID)(ownerSID), (*windows.SID)(groupSID), nil, nil)
 }
 
+func (f *BasicFilesystem) Remove(name string) error {
+	name, err := f.rooted(name)
+	if err != nil {
+		return err
+	}
+	err = os.Remove(name)
+	if os.IsPermission(err) {
+		// Try to remove the read-only attribute and try again
+		if os.Chmod(name, 0600) == nil {
+			err = os.Remove(name)
+		}
```

Comment:
`Chmod` doesn't do much on windows, right? Otherwise in similar places we'd try to re-instate the original permis if the operation/remove fails.

Reply:
It does the needful in this case, at least. I considered the whole stat and reinstate dance, but as you say this doesn't really do much on Windows, at most it removes the read only bit as the other bits and the group/other stuff is just ignored. I think simpler is better.

Reply:
FTR; Chmod on Windows only touches the read only bit based on the `0200` bit in the input:

https://github.com/golang/go/blob/8ec1c9e2850c642e3ebd0ea3a9749dd13cd4545a/src/syscall/syscall_windows.go#L647-L662